pkg/resourceinterpreter/defaultinterpreter/default.go

```diff
@@ -25,7 +25,7 @@ type DefaultInterpreter struct {
 }
 
 // NewDefaultInterpreter return a new DefaultInterpreter.
-func NewDefaultInterpreter() *DefaultInterpreter {
+func NewDefaultInterpreter(_ genericmanager.SingleClusterInformerManager) (framework.Interpreter, error) {
```

Comment:
This is one of the side effects that abstract a `framework` that some of the implementers might require different stuff.